packages/slice-machine/src/features/onboarding/OnboardingStepDialog/OnboardingStepDialog.module.css

```diff
@@ -0,0 +1,12 @@
+.content {
+  padding: 16px;
+  display: flex;
+  flex-direction: column;
+  gap: 16px;
+}
+
+.contentHeader {
+  > *:first-child {
+    margin-bottom: 4px;
+  }
+}
```

Comment:
I get the confusion. I don't think I'll use a header, because in really it's not really a header, but I changed it to apply the bottom margin directly to the element that need it instead of having a parent.